sci-geosciences/qmapshack/qmapshack-1.17.1_p20250511.ebuild

```diff
@@ -5,11 +5,11 @@ EAPI=8
 
 inherit cmake xdg
 
-COMMIT="948ae8a8a1526766ebdf9df2f207462c09c15c3b"
+COMMIT="5fc1cb76091d80094a3c3d1d50948089dd47f402"
 
 DESCRIPTION="GPS mapping utility"
 HOMEPAGE="https://github.com/Maproom/qmapshack/wiki"
-SRC_URI="https://github.com/Maproom/${PN}/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+SRC_URI="https://github.com/Maproom/${PN}/archive/${COMMIT}.tar.gz -> ${PF}.tar.gz"
 S="${WORKDIR}"/${PN}-${COMMIT}
```

Comment:
How about this instead:

```
COMMIT="5fc1cb76091d80094a3c3d1d50948089dd47f402"
MY_P=${PN}-${COMMIT}

DESCRIPTION="GPS mapping utility"
HOMEPAGE="https://github.com/Maproom/qmapshack/wiki"
SRC_URI="https://github.com/Maproom/${PN}/archive/${COMMIT}.tar.gz -> ${MY_P}.tar.gz"
S="${WORKDIR}/${MY_P}"
```

It's a bit strange to have the revision number in the name of the distfile.

Reply:
Sure, that was a leftover from a try where the commit did not change. Will push again, using a new date and commit.

Reply:
I will go back to P like it was before in the older ones. And will for now stay with the commit sha i last proposed.

Let us please keep this to "fixing QtHelp" and not endlessly improving the ebuild, that can be done later and maybe by the maintainer.

Reply:
> I will go back to P like it was before in the older ones. And will for now stay with the commit sha i last proposed.
> 
> Let us please keep this to "fixing QtHelp" and not endlessly improving the ebuild, that can be done later and maybe by the maintainer.

That's not possible, you're now changing the file we are downloading but keeping the name. This will cause manifest mismatches on the user end so we cannot merge in this state.

So it cannot be `P` and it should also not be `PF`. `PN-COMMIT` is the correct way to do it.

Reply:
i think the diffs just look funny, maybe because i used "git mv" at some place

But the first commit created a new file and its manifest entry (i even re-created it with ebuild digest)

The second one removes the old ones.

The sum looks like a "mv" and "edit". It is just the looks and some weirdness of git.

Please check again or tell me in more detail where the problem would be.

Reply:
Aah, yes you are right, sorry.